contracts/src/v0.8/ccip/libraries/Client.sol

```diff
@@ -51,9 +54,22 @@ library Client {
     bool allowOutOfOrderExecution;
   }
 
+  struct SVMExtraArgsV1 {
```

Comment:
I'm not an expert on onchain, but shouldn't this type match the onchain type used in Solana as per here:
**https://github.com/smartcontractkit/chainlink-ccip/blob/679d6f555e8dc295f92f2f385ce01ba9746968be/chains/solana/contracts/programs/ccip-router/src/messages.rs#L45-L49**

Or is the expectation that Solana onchain code will be later changed to match this?

Reply:
We're going with SVM over Sol or Solana, the Solana team still needs to change it. They should match